agents/unified/mcp_a2a_mojo_integration.py

```
@@ -115,68 +122,70 @@ async def send(self, message: UnifiedMessage, sender_pid: int, receiver_pid: int
         self.transport_stats[strategy.value]["count"] += 1
         self.transport_stats[strategy.value]["total_latency_ms"] += latency_ms
-        
+
         result["transport_latency_ms"] = latency_ms
         result["strategy"] = strategy.value
-        
+
         return result
-    
+
     async def _zero_copy_send(self, message: UnifiedMessage) -> Dict[str, Any]:
```

Ruff (0.11.9)

Unused method argument: `message`

(ARG002)